src/components/FilterTop.js

```diff
@@ -214,7 +214,7 @@ function Filter(props) {
                   format={getDateFormatLocale(true)}
                   value={dateStart}
                   onChange={handleDateStartChange}
-                  maxDate={dateEnd}
+                  //maxDate={dateEnd} Conrado Banegas comment 6/2021 issue #94
```

Comment:
A few comments:

1. We shouldn't remove `maxDate`, but should set it to today's date if `dateEnd` is `null` or `undefined`.
2. The same problem exists with `minDate` on the End Date `KeyboardDatePicker` (line 229). When `dateStart` is `null` or `undefined`, this should be set to a sensible minimum (e.g., `Date(1900-01-01)`).
3. If code is not needed, don't comment it out – remove it. We can use git to see what has been removed and put it back in if necessary.
4. There's no need to add comments like this to the code. After a while they get in the way of the code and cause the files to bloat. GitHub tracks line-by-line changes for us and we link commits to issues via PRs.

Reply:
Hi Nick.
It' ok, I'll try to develop another alternativa to solve the issue.
Thanks.